translator/trace/spandata/protospan_to_spandata.go

```diff
@@ -129,12 +131,19 @@ func protoLinkTypeToOCLinkType(lt tracepb.Span_Link_Type) trace.LinkType {
 	}
 }
 
-func protoAttributesToOCAttributes(attrs *tracepb.Span_Attributes) map[string]interface{} {
+func protoAttributesToOCAttributes(attrs *tracepb.Span_Attributes, resource *resourcepb.Resource) map[string]interface{} {
 	if attrs == nil {
```

Comment:
if there are no other span attributes but there are resources they will be missed - granted it is a corner case but possible.